### PR TITLE
Fixes logo on sticky nav

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_book: Need two thumbs, easy one - just need logo to disappear before it gets run over by search box

### CODE REVIEW

**Comment by ericanaughton** [comment]
:+1: nice work!

File: css/style.css [hunk 1]
```diff
@@ -73,14 +73,16 @@ nav.sticky-nav-color {
...
 }
 
-.fixed .nav-inner-most {
-  background: url('/img/lmht-logo.png') no-repeat;
-  display: block;
-  background-position: 10px 0px;
-  background-size: 200px 50px;
-  margin-left: 28px;
-  background-position: left;
+@media all and (min-width: 1140px) {
```
Comment (bigtiger) [inline css/style.css:76]: Not a blocker but I would assume `all` would mean we do no need `(min-width: 1140px)`.

**Comment by bigtiger** [comment]
:+1:

File: css/style.css [hunk 2]
```diff
@@ -73,14 +73,16 @@ nav.sticky-nav-color {
...
+  .fixed .nav-inner-most {
+    background: url('/img/lmht-logo.png') no-repeat;
+    display: block;
+    background-position: 10px 0px;
+    background-size: 200px 50px;
+    margin-left: 28px;
+    background-position: left;
```
Comment (missingdink) [inline css/style.css:83]: From the here on out, please sort your css in alphabetical order. Ie. all the backgrounds should be at top. Makes it easier to look at the code for other developers.

**Comment by missingdink** [comment]
:+1:  as per comment

**Comment by kathyoleary** [comment]
@bigtiger If I remove the all, I lose the logo totally, if I remove the (min-width) etc I lose the logo disappearing when I want it to disappear.  I do not know if there are any other options I can manipulate.  I can research some more or if you have some suggestions I will be happy to test them out but right now this does put the logo in and keeps it there right up to the point of the search bar infringing on the logo and then it removes it.

**Pull request merged**